types/rdf-validate-shacl/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
Why was dom added?

Reply:
Because these tests now indirectly refer to `@rdfjs/fetch-lite` which uses types from global `fetch` interface which is still only defined in the browser env